File: OCH_Tao/ABC/A/017.py
N1 = list(map(int, input().split()))
N2 = list(map(int, input().split()))
N3 = list(map(int, input().split()))
score = 0
score += int(N1[0] * N1[1] / 10)
score += int(N2[0] * N2[1] / 10)
score += int(N3[0] * N3[1] / 10)
print(score)

File: OCH_Tao/ABC/A/017.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").split("\n");
const S1 = input[0].split(" ");
const S2 = input[1].split(" ");
const S3 = input[2].split(" ");
let score = 0
score += parseInt(S1[0], 10) * parseInt(S1[1], 10) / 10;
score += parseInt(S2[0], 10) * parseInt(S2[1], 10) / 10;
score += parseInt(S3[0], 10) * parseInt(S3[1], 10) / 10;
console.log(score);